### PR TITLE
Fix + Refact

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Forms/AutoBreakUnbreakLines.cs [hunk 1]
```diff
@@ -138,12 +139,9 @@ private void Unbreak()
             listViewFixes.Items.Clear();
             foreach (Paragraph p in _paragraphs)
             {
-                if (p.Text != null && p.Text.Contains(Environment.NewLine) && p.Text.Length > minLength)
+                if (p.Text != null && p.Text.Contains(Environment.NewLine) && HtmlUtil.RemoveHtmlTags(p.Text).Length > minLength)
```
Comment (ivandrofly) [inline src/Forms/AutoBreakUnbreakLines.cs]: Shouldn't count with HTML tags

File: src/Forms/AutoBreakUnbreakLines.cs [hunk 2]
```diff
@@ -171,7 +169,7 @@ private void AddToListView(Paragraph p, string newText)
             item.SubItems.Add(subItem);
             subItem = new ListViewItem.ListViewSubItem(item, p.Text.Replace(Environment.NewLine, Configuration.Settings.General.ListViewLineSeparatorString));
             item.SubItems.Add(subItem);
-            subItem = new ListViewItem.ListViewSubItem(item, newText.Replace(Environment.NewLine, Configuration.Settings.General.ListViewLineSeparatorString));
+            subItem = new ListViewItem.ListViewSubItem(item, newText);
```
Comment (ivandrofly) [inline src/Forms/AutoBreakUnbreakLines.cs]: `\r\n` will be removed before getting here

File: src/Logic/Utilities.cs [hunk 3]
```diff
@@ -792,14 +792,25 @@ private static string ReInsertHtmlTags(string s, Dictionary<int, string> htmlTag
...
-                return newText;
+                singleLine = singleLine.Replace("</i> <i>", " ");
+                singleLine = singleLine.Replace("</i><i>", " ");
+
+                singleLine = singleLine.Replace("</b> <b>", " ");
+                singleLine = singleLine.Replace("</b><b>", " ");
+
+                singleLine = singleLine.Replace("</u> <u>", " ");
+                singleLine = singleLine.Replace("</u><u>", " ");
             }
```
Comment (ivandrofly) [inline src/Logic/Utilities.cs]: Useful when *user* just want to *unbreak* line without running the **Fix Common Errors**

File: src/Forms/AutoBreakUnbreakLines.cs (hunk 2 above)
Comment (niksedk) [inline src/Forms/AutoBreakUnbreakLines.cs]: I don't think so... also tested it ;)

Comment (ivandrofly) [inline src/Forms/AutoBreakUnbreakLines.cs]: :+1:, test I did was only for UnbreakLines, Reverted

Comment (ivandrofly) [inline src/Forms/AutoBreakUnbreakLines.cs]: > I don't think so... also tested it ;)

:+1:, test I did was only for UnbreakLines, Reverted

**Comment by ivandrofly** [comment]
Should

**Comment by ivandrofly** [comment]
https://github.com/SubtitleEdit/subtitleedit/pull/895

**Pull request closed without merging**